vmb/matrimony/models/profiles.py

```diff
@@ -335,6 +336,16 @@ class MatrimonyProfile(BaseModel):
         related_name="assigned_profiles",
     )
 
+    @property
+    def primary_image(self):
+        if self.images is not None and self.images != "":
+            return format_html(
+                '<img src ="{}" style="width:30px; \
+                height: 30px"/>'.format(
+                    Image.objects.get(profile=self, primary=True).photo.image.url
```

Comment:
This will break if no image is primary or no images are available.

Reply:
it does not break if no image is available, I will check the primary part and add try, except

Reply:
A better way to query is `self.images...`